anubis-management-api/src/wac.py

```diff
@@ -3,21 +3,31 @@
 from rdflib.namespace import FOAF, RDF
 from policies.models import Policy
 from sqlalchemy.orm import Session
+import yaml
+import os
 
-n = Namespace("http://example.org/")
 acl = Namespace("http://www.w3.org/ns/auth/acl#")
 
-
 # follow keycloak scheme
 # user urls should be /{realm}/users/{id}
 # group urls should be /{realm}/groups/{id}
 
-def serialize(db: Session, policies: [Policy]):
+
+def serialize(db: Session, fiware_service: [str], policies: [Policy]):
+    with open(os.environ.get("DEFAULT_WAC_CONFIG_FILE", '../../config/opa-service/default_wac_config.yml'), 'r') as file:
```

Comment:
this env variable is missing in the readme and in the install
https://github.com/orchestracities/anubis/blob/master/docs/admin/install.md